site/src/App/Navigation/Navigation.tsx

```diff
@@ -105,7 +105,9 @@ const PreviewBranchPanel = () => {
             )}
           >
             <MenuItemLink
-              href={`https://seek-oss.github.io/braid-design-system${pathname}/${hash}`}
+              href={`https://seek-oss.github.io/braid-design-system${pathname}${
+                !pathname.endsWith('/') ? '/' : ''
```

Comment:
`/` isn't necessary before the has right? Can't this just be removed altogether?

Reply:
Yeah thats what i thought! I initially had no slash, but the hash wasnt appending to the link. Let me test is again, maybe my dev server was in a weird state.